CodeForAfricaLabs/CitizenReporter.Android
# Add Push Notification Feature

#### What does this PR do?
Allow users to receive push notification.
#### Description of Task to be completed?
Send push notifications to all clients(messenger, android, iOS) consuming the API on creation of a new assignment.
#### How should this be manually tested?
After cloning the repo and running the app on Android studio:
- Ensure the app is running on the background, go to the logcat on Android studio and copy the registration token displayed
- Go to the Firebase Cloud Messenger: https://console.firebase.google.com/project/storied-shell-179008/overview
- Go to the Notifications section, click on send message, enter any message text, then paste the registration token to the single device target.
- Click Send Message button. You should receive a notification on your Android device with the message you typed on the Firebase Cloud Messenger console
#### Any background context you want to provide?
Make sure you device has google playstore installed.

<img width="447" alt="screen shot 2017-09-08 at 17 09 41" src="https://user-images.githubusercontent.com/25563841/30215433-a5f44a12-94b8-11e7-9645-009cf084b5e5.png">
